detd/scheduler.py

```diff
@@ -389,3 +385,53 @@ def reschedule(self):
 
             self.schedule.add_best_effort_padding(self.traffics[0])
             # FIXME: error handling
+
+class TxSelection(Enum):
+    ENHANCEMENTS_FOR_SCHEDULED_TRAFFIC = "802.1Qbv"
```

Comment:
I guess we can use numbers for the values? If we need them to be explicit for the protobuf interface, that may be a good idea.

Otherwise, instead of providing the strings directly, maybe auto() can be used instead, that I understand would have a similar effect.